src/test/java/org/commonwl/view/workflow/QueuedWorkflowRepositoryTest.java

```diff
@@ -5,47 +5,35 @@
 import static org.junit.jupiter.api.Assertions.assertNull;
 
 import java.util.List;
-import org.commonwl.view.CwlViewerApplication;
-import org.commonwl.view.WebConfig;
 import org.commonwl.view.git.GitDetails;
 import org.junit.jupiter.api.Test;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.boot.autoconfigure.domain.EntityScan;
 import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase;
 import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
-import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
-import org.springframework.test.context.ActiveProfiles;
 import org.springframework.test.context.ContextConfiguration;
-import org.springframework.transaction.annotation.Propagation;
+import org.springframework.test.context.TestPropertySource;
 import org.springframework.transaction.annotation.Transactional;
-import org.testcontainers.containers.PostgreSQLContainer;
-import org.testcontainers.junit.jupiter.Container;
-import org.testcontainers.junit.jupiter.Testcontainers;
 
-@ActiveProfiles("it")
-@Testcontainers
+@TestPropertySource(locations = "classpath:it-application.properties")
 // @SpringBootTest(
 //        classes={WebConfig.class, CwlViewerApplication.class, QueuedWorkflowRepository.class}
 // )
 @DataJpaTest(showSql = true)
-@EnableJpaRepositories
-@EntityScan
-@Transactional(propagation = Propagation.REQUIRED)
+@ContextConfiguration(initializers = PostgreSQLContextInitializer.class)
+// @EnableJpaRepositories("org.commonwl.view.workflow")
+// @EntityScan
+// @Transactional(propagation = Propagation.REQUIRED)
+// @ContextConfiguration(
+//    initializers = PostgreSQLContextInitializer.class,
+//    classes = {WebConfig.class, CwlViewerApplication.class, QueuedWorkflowRepository.class})
+// @SpringBootTest(classes = CwlViewerApplication.class)
+// @RunWith(SpringRunner.class)
```

Comment:
What's the story here?

Reply:
I used this to test, but I can remove this. I don't think I, or any other developer, will need this. Removing it...

Reply:
Sorry, got distracted by something. Done, and all tests passed.